packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -73,7 +84,7 @@ if (process.env.NODE_ENV === 'development') {
   BreadcrumbItem.propTypes = {
     children: PropTypes.node.isRequired,
     element: PropTypes.string,
-    href: PropTypes.string,
+    href: PropTypes.string.isRequired,
```

Comment:
Is this a required prop? On line 57 it looks like it's possible it is undefined.

Reply:
It throws an error if we don't add `.isRequired` that looks like this:
```
Type 'Requireable<string>' is not assignable to type 'Validator<string>'.
  Types of property '[nominalTypeHack]' are incompatible.
    Type '{ type: string | null | undefined; } | undefined' is not assignable to type '{ type: string; } | undefined'.
      Type '{ type: string | null | undefined; }' is not assignable to type '{ type: string; }'.
        Types of property 'type' are incompatible.
          Type 'string | null | undefined' is not assignable to type 'string'.
            Type 'undefined' is not assignable to type 'string'.ts(2322)
```

I'm not exactly sure what to make of it so just went with it tbh

Reply:
Actually, this is kind of tricky on second thought. React proptypes is trying to infer the type based on the typescript types. But it can't understand that it is conditional, it's likely taking the last one it sees, or when it is an anchor and required, it trumps everything. I naively just made it required to satisfy the compiler, but thats not accurate at all.

You might have to remove it entirely, or `@ts-expect-error` as we're still figuring out conditionally typed refs

Reply:
Looking at [Badge prop types](https://github.com/twilio-labs/paste/blob/91a8a4943a6aa7ec0c17619ff662892b89c22cc6/packages/paste-core/components/badge/src/index.tsx#L64), we're able to say `href` isn't required when it is required sometimes...